cmake/FindGemmTools.cmake

```diff
@@ -49,6 +49,12 @@ foreach(component ${_GEMM_TOOLS_LIST})
     elseif ("${component}" STREQUAL "Eigen")
         # already included by default!
 
+    elseif ("${component}" STREQUAL "GemmForge")
+        execute_process(COMMAND python -c "import gemmforge; gemmforge.print_cmake_path()"
```

Comment:
It would be better style to use https://cmake.org/cmake/help/v3.16/module/FindPython.html for this and specify that we want to have python3. Your version would use python2 on most systems. (I assume gemmforge is python3, as python2 is basically dead)

Reply:
Ideally, we need to check a presence of python3 at top of the main CMake file.